app/scripts/lib/bloomfilter.js

```diff
@@ -0,0 +1,118 @@
+define([
+],
+function() {
+  'use strict';
+  var typedArrays = typeof ArrayBuffer !== "undefined";
```

Comment:
bloomfilter.js could be its own repository with tests and then included via bower.json